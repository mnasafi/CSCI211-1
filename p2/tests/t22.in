length
title 00
url 00
comment 00
43.96
4
title 01
url 01
comment 01
26.13
2
title 02
url 02
comment 02
75.3
2
title 03
url 03
comment 03
60.79
3
title 04
url 04
comment 04
17.47
3
title 05
url 05
comment 05
26.28
4
title 06
url 06
comment 06
64.44
3
title 07
url 07
comment 07
90.86
1
title 08
url 08
comment 08
71.25
2
title 09
url 09
comment 09
57.10
1
title 10
url 10
comment 10
54.61
5
title 11
url 11
comment 11
67.62
1
title 12
url 12
comment 12
22.38
5
title 13
url 13
comment 13
2.87
1
title 14
url 14
comment 14
1.84
3
title 15
url 15
comment 15
29.96
5
title 16
url 16
comment 16
74.81
3
title 17
url 17
comment 17
12.64
2
title 18
url 18
comment 18
37.36
4
title 19
url 19
comment 19
99.7
1
title 20
url 20
comment 20
61.69
4
title 21
url 21
comment 21
75.73
2
title 22
url 22
comment 22
65.55
3
title 23
url 23
comment 23
52.11
4
title 24
url 24
comment 24
88.27
2
title 25
url 25
comment 25
85.11
5
title 26
url 26
comment 26
18.20
5
title 27
url 27
comment 27
82.44
1
title 28
url 28
comment 28
70.58
4
title 29
url 29
comment 29
77.93
5
title 30
url 30
comment 30
46.33
2
title 31
url 31
comment 31
72.78
1
title 32
url 32
comment 32
79.41
5
title 33
url 33
comment 33
91.74
2
title 34
url 34
comment 34
18.59
4
title 35
url 35
comment 35
29.45
5
title 36
url 36
comment 36
1.12
5
title 37
url 37
comment 37
97.3
3
title 38
url 38
comment 38
7.6
3
title 39
url 39
comment 39
1.44
4
title 40
url 40
comment 40
34.33
2
title 41
url 41
comment 41
65.86
5
title 42
url 42
comment 42
6.61
3
title 43
url 43
comment 43
32.1
5
title 44
url 44
comment 44
44.44
2
title 45
url 45
comment 45
41.74
2
title 46
url 46
comment 46
53.41
4
title 47
url 47
comment 47
9.25
1
title 48
url 48
comment 48
15.42
2
title 49
url 49
comment 49
11.29
4
title 50
url 50
comment 50
97.93
3
title 51
url 51
comment 51
83.6
4
title 52
url 52
comment 52
45.90
4
title 53
url 53
comment 53
98.52
3
title 54
url 54
comment 54
94.9
1
title 55
url 55
comment 55
68.30
1
title 56
url 56
comment 56
9.84
1
title 57
url 57
comment 57
34.89
1
title 58
url 58
comment 58
77.91
5
title 59
url 59
comment 59
6.0
2
title 60
url 60
comment 60
51.8
3
title 61
url 61
comment 61
57.2
3
title 62
url 62
comment 62
99.60
4
title 63
url 63
comment 63
4.76
1
title 64
url 64
comment 64
13.23
4
title 65
url 65
comment 65
95.65
3
title 66
url 66
comment 66
31.47
2
title 67
url 67
comment 67
21.34
2
title 68
url 68
comment 68
64.29
3
title 69
url 69
comment 69
64.90
1
title 70
url 70
comment 70
73.62
3
title 71
url 71
comment 71
27.81
2
title 72
url 72
comment 72
88.35
5
title 73
url 73
comment 73
16.33
5
title 74
url 74
comment 74
91.80
2
title 75
url 75
comment 75
8.20
4
title 76
url 76
comment 76
55.80
4
title 77
url 77
comment 77
42.96
4
title 78
url 78
comment 78
71.97
3
title 79
url 79
comment 79
14.27
3
title 80
url 80
comment 80
28.14
2
title 81
url 81
comment 81
9.51
4
title 82
url 82
comment 82
97.97
1
title 83
url 83
comment 83
82.94
3
title 84
url 84
comment 84
63.93
3
title 85
url 85
comment 85
35.57
5
title 86
url 86
comment 86
15.15
1
title 87
url 87
comment 87
63.3
2
title 88
url 88
comment 88
61.64
5
title 89
url 89
comment 89
40.88
3
title 90
url 90
comment 90
54.75
5
title 91
url 91
comment 91
5.39
5
title 92
url 92
comment 92
55.77
1
title 93
url 93
comment 93
49.54
3
title 94
url 94
comment 94
94.46
2
title 95
url 95
comment 95
51.28
3
title 96
url 96
comment 96
18.69
3
title 97
url 97
comment 97
22.64
5
title 98
url 98
comment 98
38.1
2
title 99
url 99
comment 99
26.60
3
