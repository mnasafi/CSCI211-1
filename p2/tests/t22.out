title 36, url 36, comment 36, 1.12, *****
title 39, url 39, comment 39, 1.44, ****
title 14, url 14, comment 14, 1.84, ***
title 13, url 13, comment 13, 2.87, *
title 63, url 63, comment 63, 4.76, *
title 91, url 91, comment 91, 5.39, *****
title 59, url 59, comment 59, 6, **
title 42, url 42, comment 42, 6.61, ***
title 38, url 38, comment 38, 7.6, ***
title 75, url 75, comment 75, 8.2, ****
title 47, url 47, comment 47, 9.25, *
title 81, url 81, comment 81, 9.51, ****
title 56, url 56, comment 56, 9.84, *
title 49, url 49, comment 49, 11.29, ****
title 17, url 17, comment 17, 12.64, **
title 64, url 64, comment 64, 13.23, ****
title 79, url 79, comment 79, 14.27, ***
title 86, url 86, comment 86, 15.15, *
title 48, url 48, comment 48, 15.42, **
title 73, url 73, comment 73, 16.33, *****
title 04, url 04, comment 04, 17.47, ***
title 26, url 26, comment 26, 18.2, *****
title 34, url 34, comment 34, 18.59, ****
title 96, url 96, comment 96, 18.69, ***
title 67, url 67, comment 67, 21.34, **
title 12, url 12, comment 12, 22.38, *****
title 97, url 97, comment 97, 22.64, *****
title 01, url 01, comment 01, 26.13, **
title 05, url 05, comment 05, 26.28, ****
title 99, url 99, comment 99, 26.6, ***
title 71, url 71, comment 71, 27.81, **
title 80, url 80, comment 80, 28.14, **
title 35, url 35, comment 35, 29.45, *****
title 15, url 15, comment 15, 29.96, *****
title 66, url 66, comment 66, 31.47, **
title 43, url 43, comment 43, 32.1, *****
title 40, url 40, comment 40, 34.33, **
title 57, url 57, comment 57, 34.89, *
title 85, url 85, comment 85, 35.57, *****
title 18, url 18, comment 18, 37.36, ****
title 98, url 98, comment 98, 38.1, **
title 89, url 89, comment 89, 40.88, ***
title 45, url 45, comment 45, 41.74, **
title 77, url 77, comment 77, 42.96, ****
title 00, url 00, comment 00, 43.96, ****
title 44, url 44, comment 44, 44.44, **
title 52, url 52, comment 52, 45.9, ****
title 30, url 30, comment 30, 46.33, **
title 93, url 93, comment 93, 49.54, ***
title 95, url 95, comment 95, 51.28, ***
title 60, url 60, comment 60, 51.8, ***
title 23, url 23, comment 23, 52.11, ****
title 46, url 46, comment 46, 53.41, ****
title 10, url 10, comment 10, 54.61, *****
title 90, url 90, comment 90, 54.75, *****
title 92, url 92, comment 92, 55.77, *
title 76, url 76, comment 76, 55.8, ****
title 09, url 09, comment 09, 57.1, *
title 61, url 61, comment 61, 57.2, ***
title 03, url 03, comment 03, 60.79, ***
title 88, url 88, comment 88, 61.64, *****
title 20, url 20, comment 20, 61.69, ****
title 87, url 87, comment 87, 63.3, **
title 84, url 84, comment 84, 63.93, ***
title 68, url 68, comment 68, 64.29, ***
title 06, url 06, comment 06, 64.44, ***
title 69, url 69, comment 69, 64.9, *
title 22, url 22, comment 22, 65.55, ***
title 41, url 41, comment 41, 65.86, *****
title 11, url 11, comment 11, 67.62, *
title 55, url 55, comment 55, 68.3, *
title 28, url 28, comment 28, 70.58, ****
title 08, url 08, comment 08, 71.25, **
title 78, url 78, comment 78, 71.97, ***
title 31, url 31, comment 31, 72.78, *
title 70, url 70, comment 70, 73.62, ***
title 16, url 16, comment 16, 74.81, ***
title 02, url 02, comment 02, 75.3, **
title 21, url 21, comment 21, 75.73, **
title 58, url 58, comment 58, 77.91, *****
title 29, url 29, comment 29, 77.93, *****
title 32, url 32, comment 32, 79.41, *****
title 27, url 27, comment 27, 82.44, *
title 83, url 83, comment 83, 82.94, ***
title 51, url 51, comment 51, 83.6, ****
title 25, url 25, comment 25, 85.11, *****
title 24, url 24, comment 24, 88.27, **
title 72, url 72, comment 72, 88.35, *****
title 07, url 07, comment 07, 90.86, *
title 33, url 33, comment 33, 91.74, **
title 74, url 74, comment 74, 91.8, **
title 94, url 94, comment 94, 94.46, **
title 54, url 54, comment 54, 94.9, *
title 65, url 65, comment 65, 95.65, ***
title 37, url 37, comment 37, 97.3, ***
title 50, url 50, comment 50, 97.93, ***
title 82, url 82, comment 82, 97.97, *
title 53, url 53, comment 53, 98.52, ***
title 62, url 62, comment 62, 99.6, ****
title 19, url 19, comment 19, 99.7, *
